unit-tests/live/rec-play/test-playback-stress.py

```diff
@@ -0,0 +1,70 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2021 Intel Corporation. All Rights Reserved.
+
+#test:timeout 1500
+#test:donotrun:!nightly
+
+import pyrealsense2 as rs, os, time
+from rspy import log, test, repo
+
+file_name = os.path.join( repo.build, 'unit-tests', 'recordings', 'all_combinations_depth_color.bag' )
+log.d( 'deadlock file:', file_name )
+frames_in_bag_file = 32
+
+frames_count = 0
+
+def frame_callback():
+    global frames_count
+    frames_count += 1
+
+
+################################################################################################
+test.start( "Playback stress test" )
+
+log.d( "Playing back: " + file_name )
+for i in range(250):
+    try:
+        log.d("Starting iteration # " , i)
+        ctx = rs.context()
+        dev = ctx.load_device( file_name )
+        dev.set_real_time( True )
+        sensors = dev.query_sensors()
+        frames_count = 0
+        for sensor in sensors:
+            sensor.open( sensor.get_stream_profiles() )
+            
+        for sensor in sensors:
+            sensor.start( frame_callback() )
+    
+        while( True ):
```

Comment:
This will run forever if it never starts playing...

Reply:
In any case, please explain

Reply:
I am counting on the test timeout for it.
I will change it to a timer that is higher than the test timeout.